app/pages/lab/workflow.cjsx

```diff
@@ -159,6 +160,15 @@ EditWorkflowPage = React.createClass
                         <small><strong>Crop</strong></small>
                       </button>
                     </AutoSave>}{' '}
+
+                  {if @canUseTask(@props.project, "selection")
```

Comment:
Mind adding "selection" to the list of experimental features in pages/admin/project-status?